pkg/webmethods/client.go

```diff
@@ -440,7 +442,7 @@ func (c *WebMethodClient) GetStrategy(strategyId string) (*StrategyResponse, err
 }
 
 func (c *WebMethodClient) SubscribeApplication(applicationId string, ApplicationApiSubscription *ApplicationApiSubscription) error {
-	url := fmt.Sprintf("%s/rest/apigateway/applications/%s/apis", c.url, applicationId)
+	url := fmt.Sprintf(getApplicationURL+"/apis", c.url, applicationId)
```

Comment:
why do you have + signs on some of these

Reply:
I guess, otherwise you will need separate fmt template for each applicationURL, applicationURL/apis, applicationURL/accessTokens